### DISCUSSION TITLE
Can I change ANSI colors in the termux terminal?

### DISCUSSION THREAD

**Discussion by mtatton** [termux/termux-packages#10877] [Q&A] [answered]
Hi, can I change colors in the termux terminal like this?
[HOWTO Configure XTerm Colors as PuTTY](https://bbst.neocities.org/doc/howto_xterm_as_putty.html)

**Comment by SherkeyXD** [comment, marked as answer]
try this https://github.com/Cabbagec/termux-ohmyzsh

**Reply by mtatton** [reply]
~/.termux $ cat colors.properties # VGA ANSI GRAPHICS TERMUX SETTINGS boldColors=on foreground=#ffffff background=#000000 cursor=#a0a0a0 color0=#000000 color1=#aa0000 color2=#00aa00 color3=#aa5500 color4=#0000aa color5=#aa00aa color6=#00aaaa color7=#aaaaaa color8=#555555 color9=#ff5555 color10=#55ff55 color11=#ffff55 color12=#5555ff color13=#ff55ff color14=#55ffff color15=#ffffff
